<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_temp">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_resultat">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="chiffre" name="pushButton_1">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="chiffre" name="pushButton_4">
                <property name="text">
                 <string>4</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="chiffre" name="pushButton_7">
                <property name="text">
                 <string>7</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="chiffre" name="pushButton_2">
                <property name="text">
                 <string>2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="chiffre" name="pushButton_5">
                <property name="text">
                 <string>5</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="chiffre" name="pushButton_8">
                <property name="text">
                 <string>8</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="chiffre" name="pushButton_3">
                <property name="text">
                 <string>3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="chiffre" name="pushButton_6">
                <property name="text">
                 <string>6</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="chiffre" name="pushButton_9">
                <property name="text">
                 <string>9</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="chiffre" name="pushButton_0">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_egale">
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_C">
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="operateur" name="pushButton_plus">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="operateur" name="pushButton_moins">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="operateur" name="pushButton_multiplier">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="operateur" name="pushButton_diviser">
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>372</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>chiffre</class>
   <extends>QPushButton</extends>
   <header>chiffre.h</header>
  </customwidget>
  <customwidget>
   <class>operateur</class>
   <extends>QPushButton</extends>
   <header>operateur.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
